docker-compose-prod.yml

```diff
@@ -0,0 +1,51 @@
+version: '2'
+services:
+  nginx:
+    image: jwilder/nginx-proxy
+    ports: 
+      - "80:80"
+      - "443:443"
+    volumes:
+      - /var/run/docker.sock:/tmp/docker.sock:ro
+  server:
+    container_name: app_stt
+    build:
+      context: .
+      dockerfile: ./docker/Dockerfile-server-dev
```

Comment:
So does that mean that we don't need to do a git pull?  It's not pulling code from the local repo?